ambry-api/src/main/java/com/github/ambry/config/MySqlAccountServiceConfig.java

```diff
@@ -0,0 +1,94 @@
+ */
+package com.github.ambry.config;
+
+/**
+ * Config for {@link MySqlAccountServiceConfig}
+ */
+public class MySqlAccountServiceConfig {
+  public static final String MYSQL_ACCOUNT_SERVICE_PREFIX = "mysql.account.service.";
+  public static final String DB_URL = MYSQL_ACCOUNT_SERVICE_PREFIX + "db.url";
+  public static final String DB_USER = MYSQL_ACCOUNT_SERVICE_PREFIX + "db.user";
+  public static final String DB_PASSWORD = MYSQL_ACCOUNT_SERVICE_PREFIX + "db.password";
+  public static final String UPDATER_POLLING_INTERVAL_SECONDS =
+      MYSQL_ACCOUNT_SERVICE_PREFIX + "updater.polling.interval.seconds";
+  public static final String UPDATER_SHUT_DOWN_TIMEOUT_SECONDS =
+      MYSQL_ACCOUNT_SERVICE_PREFIX + "updater.shut.down.timeout.seconds";
+  public static final String BACKUP_DIRECTORY_KEY = MYSQL_ACCOUNT_SERVICE_PREFIX + "backup.dir";
+  public static final String UPDATE_DISABLED = MYSQL_ACCOUNT_SERVICE_PREFIX + "update.disabled";
+
+  // TODO: Might need to take an array of URLs which would have one write (master) and multiple read urls (backup)
+  /**
+   * URL of the mysql database.
+   */
+  @Config(DB_URL)
+  @Default("")
+  public final String dbUrl;
+
+  /**
+   * Username for the mysql database.
+   */
+  @Config(DB_USER)
+  @Default("")
+  public final String dbUser;
+
+  /**
+   * Password for the mysql database.
+   */
+  @Config(DB_PASSWORD)
+  @Default("")
+  public final String dbPassword;
+
+  /**
+   * The time interval in seconds between two consecutive account pulling for the background account updater of
+   * {@code MySqlAccountService}. Setting to 0 will disable it.
+   */
+  @Config(UPDATER_POLLING_INTERVAL_SECONDS)
+  @Default("60")
+  public final int updaterPollingIntervalSeconds;
+
+  /**
+   * The timeout in seconds to shut down the account updater of {@code MySqlAccountService}.
+   */
+  @Config(UPDATER_SHUT_DOWN_TIMEOUT_SECONDS)
+  @Default("5")
+  public final int updaterShutDownTimeoutSeconds;
+
+  /**
+   * The directory on the local machine where account data backups will be stored before updating accounts.
+   * If this string is empty, backups will be disabled.
+   */
+  @Config(BACKUP_DIRECTORY_KEY)
+  @Default("")
+  public final String backupDir;
+
+  /**
+   * If true, MySqlAccountService would reject all the requests to update accounts.
+   */
+  @Config(UPDATE_DISABLED)
+  @Default("false")
+  public final boolean updateDisabled;
+
+  public MySqlAccountServiceConfig(VerifiableProperties verifiableProperties) {
+    dbUrl = verifiableProperties.getString(DB_URL);
+    dbUser = verifiableProperties.getString(DB_USER);
+    dbPassword = verifiableProperties.getString(DB_PASSWORD);
+    updaterPollingIntervalSeconds =
+        verifiableProperties.getIntInRange(UPDATER_POLLING_INTERVAL_SECONDS, 60, 0, Integer.MAX_VALUE);
+    updaterShutDownTimeoutSeconds =
+        verifiableProperties.getIntInRange(UPDATER_SHUT_DOWN_TIMEOUT_SECONDS, 5, 1, Integer.MAX_VALUE);
```

Comment:
Is 5 seconds sufficient to shutdown the scheduler in MySqlAccountService if there is an ongoing task performing "fetchAndUpdate" with Mysql?